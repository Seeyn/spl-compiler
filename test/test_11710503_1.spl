int add(int a,int b){
    return a+b;
}
int main(){
    int i = 0;
    j = 1;
    add();
}