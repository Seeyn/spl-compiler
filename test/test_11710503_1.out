Error type 1 at Line 6: undefined variable: j
Error type 9 at Line 7: expect argument